utils/docgenMarkdown/index.ts

```diff
@@ -0,0 +1,91 @@
+// BASED HEAVILY IN: https://hardhat.org/plugins/hardhat-docgen.html
```

Comment:
We need to process the .sol files (really the compilation output) and generate the markdown. This can be automated but to test the process locally you must run the command